megamek/i18n/megamek/common/report-messages.properties

```diff
@@ -1189,3 +1189,12 @@
 9982=Pulse
 9983=Missile
 9984=Burst-fire
+
+9986=Insufficient atmospheric pressure for fuel-air weapon. No damage inflicted.
+9987=Fuel-air weapon against infantry may result in automatic destruction. Needs <data>-, rolled <data> (2d6) + <data> (distance from blast center) : <msg:9989,9988>
```

Comment:
Is that `-` after the first `<data>` intended?

Reply:
Yes, unlike every other type of roll in battletech, you're trying to roll lower than the target number.

Reply:
#Groan. Good to know!